src/services/userNotification.ts

```diff
@@ -79,6 +79,29 @@ export const enableSavedSearch = async ({
   }
 }
 
+export const extendSavedSearch = async ({
+  key,
+  options = {},
+}: {
+  key: string
+  options?: ApiCallOptions
+}): Promise<WithValidationError> => {
+  try {
+    await postData({
+      path: `saved-searches/key/${key}/extend`,
+      body: {},
+      options,
+    })
+
+    return {
+      tag: "success",
+      result: {},
+    }
+  } catch (error) {
+    return handleValidationError(error, { swallowErrors: true })
```

Comment:
What kind of validation error is expected to be returned from the endpoint?

Reply:
It will return a 404

Reply:
I'm confused now. a validation error should not be a 404 not found. did you mean 400 bad request? I think Lorenzo is asking what validations this endpoint does on which fields. maybe worth that you two are getting in a quick call to clear up the question/answer

Reply:
(with the 404 you might have intended to say that if the key is wrong, it would be a 404. in that case, there would be no need to handle validation errors because there are no actual fields that get validated. the 404 should be properly raised and handled by the app imo)

Reply:
If the key is wrong it will return a 404. The same happens for `enableSavedSearch`, we have a 404 if the key is wrong but we still handle validation errors in this case, so I am confused 😕

Reply:
Marine and I discussed that it is not needed to add validation error handling in this case as the only error response sent is a 404 for an invalid key, in comparison to forms where we receive detailed information about the post body validation